rpc/src/v1/traits/eth.rs

```diff
@@ -63,6 +63,10 @@ build_rpc_trait! {
 		#[rpc(name = "eth_getBalance")]
 		fn balance(&self, H160, Trailing<BlockNumber>) -> BoxFuture<U256>;
 
+		/// Returns balance of the given account.
```

Comment:
This description is not accurate because it actually does much more than that. i.e, includes Merkle-proofs for both the account and the storage.

Why as you stated in the EIP: `Returns the account- and storage-values of the specified account including the Merkle-proof`?

Reply:
fixed